front_end/src/App.js

```diff
@@ -77,6 +77,7 @@ class App extends Component {
 
             <Route exact path="/register" component={Register} />
             <Route exact path="/login" component={Login} />
+            {/*<Route exact path="/indiv-hotel/:hotelID" component={IndivHotel} />*/}
```

Comment:
Please leave this comment. It will be needed once backend team creates a backend API that allows fetching data of one individual hotel using only its database id.

Reply:
okaaay